.github/workflows/linux-eic-shell.yml

```diff
@@ -95,7 +95,7 @@ jobs:
         platform-release: "jug_xl:nightly"
         run: |
           CC=${{ matrix.CC }} CXX=${{ matrix.CXX }} cmake -B build -S . -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
-          cmake --build build -- -j 2 install
+          cmake --build build -- -k -j 2 install
```

Comment:
What is `-k`?
https://cmake.org/cmake/help/git-master/manual/cmake.1.html
Please, use long option, if available.

Reply:
The options after `--` are passed to the build system; they are not CMake options. The build system can be either `ninja` or `make` (and I think we prefer `ninja` in CI because it is faster than `make`) Both support `-k`, but only `make` has the longer version `--keep-going`; `ninja` has no long options. The purpose of this option is essentially the opposite of "fail fast", and it keeps compiling files (that aren't dependent on failed files) so errors in as many source files as possible are made visible in a single pass.

Reply:
We don't specify `ninja` here. We can and probably should with `cmake -G ninja` (also no long form).

Reply:
My statements on relative speed of `ninja` versus `make` are not based on my own tests or experience but on what for example dd4hep uses in its source-based distribution through spack.

Reply:
So, long story short (and many words about one letter): yes, we should prefer long options always, but in this case the long option is not currently supported by one build system choice which we/dd4hep/cmake may now/later use by default.

Reply:
Just realized that myself, the `--` was a clue. I agree, we want `--keep-going` in CI.